plugins/analysis-icu/src/test/java/org/elasticsearch/index/analysis/SimpleIcuCollationTokenFilterTests.java

```diff
@@ -218,6 +218,22 @@ public void testCustomRules() throws Exception {
         TokenFilterFactory filterFactory = analysis.tokenFilter.get("myCollator");
         assertCollatesToSame(filterFactory, "Töne", "Toene");
     }
+    
+    /*
+     * Test a basic custom rules (should not interfere with reading rules list
+     * in IcuCollationTokenFilterFactory and throw InvalidPathException).
```

Comment:
Nit: since the InvalidPathException only seems to be thrown on Windows platform (the test passes on others without adding it to the "catch" in IcuCollationTokenFilterFactory), I'd either remove this part of the comment or make it more precise.